src/Migration/Reader/Map.php

```diff
@@ -336,7 +331,6 @@ protected function validateType($type)
      */
     protected function getOppositeType($type)
     {
-        $this->validateType($type);
```

Comment:
Validation can be omitted, since the method is called only by the `getFieldMap()` method, which has already done the validation.